apps/meteor/app/livechat/server/lib/RoutingManager.ts

```diff
@@ -48,7 +48,12 @@ type Routing = {
 		options?: { clientAction?: boolean; forwardingToDepartment?: { oldDepartmentId?: string; transferData?: any } },
 		room?: IOmnichannelRoom,
 	): Promise<(IOmnichannelRoom & { chatQueued?: boolean }) | null | void>;
-	unassignAgent(inquiry: ILivechatInquiryRecord, departmentId?: string, shouldQueue?: boolean): Promise<boolean>;
+	unassignAgent(
+		inquiry: ILivechatInquiryRecord,
+		departmentId?: string,
+		shouldQueue?: boolean,
+		agent?: SelectedAgent | null,
```

Comment:
is this enough? not sure I get it why could pass `null` if it was already optional
```suggestion
		agent?: SelectedAgent,
```

Reply:
https://rocketchat.atlassian.net/browse/CTZ-303

Right here